clients/admin-ui/cypress/e2e/systems-classify.cy.ts

```diff
@@ -35,5 +57,99 @@ describe("Classify systems page", () => {
       cy.visit("/classify-systems");
       cy.getByTestId("no-classifications");
     });
+
+    it("Should render a proper description based on the existence of classification and data flows", () => {
+      cy.intercept("GET", "/api/v1/plus/classify/details/*", {
```

Comment:
ahh okay I think I needed to close the drawer since it was automatically reopening after keeping the active system in redux (probably due to redux persist?)